cmd/image/import/import.go

```diff
@@ -94,11 +71,8 @@ pvsadm image import -n upstream-core-lon04 -b <BUCKETNAME>  --object rhel-83-100
 		if pkg.ImageCMDOptions.InstanceID == "" && pkg.ImageCMDOptions.InstanceName == "" {
 			return fmt.Errorf("--pvs-instance-name or --pvs-instance-id required")
 		}
-
-		case1 := pkg.ImageCMDOptions.AccessKey == "" && pkg.ImageCMDOptions.SecretKey != ""
-		case2 := pkg.ImageCMDOptions.AccessKey != "" && pkg.ImageCMDOptions.SecretKey == ""
-
-		if case1 || case2 {
+		// ensure that both, the AccessKey and SecretKey are either both set or unset
+		if (len(pkg.ImageCMDOptions.AccessKey) > 0) != (len(pkg.ImageCMDOptions.SecretKey) > 0) {
```

Comment:
will length of access key is always equal to secret key?

Reply:
The objective here is to compare that the AccessKey and SecretKey is either present or not present. The `!=` achieves the XNOR operation, which returns true only if either of them are set.

Reply:
Misread the expression,Thanks